# helpful_links

[![Check Links](https://github.com/paulstothard/helpful_links/actions/workflows/links.yml/badge.svg)](https://github.com/paulstothard/helpful_links/actions/workflows/links.yml) [![GitHub Super-Linter](https://github.com/paulstothard/helpful_links/actions/workflows/linter.yml/badge.svg)](https://github.com/paulstothard/helpful_links/actions/workflows/linter.yml) [![Generate TOC](https://github.com/paulstothard/helpful_links/actions/workflows/toc.yml/badge.svg)](https://github.com/paulstothard/helpful_links/actions/workflows/toc.yml)

Links related to bioinformatics, data analysis, and science.

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
**Table of Contents**

- [Containers](#containers)
- [Data visualization](#data-visualization)
- [Drawing](#drawing)
- [Genome annotation](#genome-annotation)
- [Genome assembly](#genome-assembly)
- [GWAS](#gwas)
- [Machine learning](#machine-learning)
- [Metagenomics](#metagenomics)
- [Phylogenomics](#phylogenomics)
- [Pipeline development](#pipeline-development)
- [Programming](#programming)
- [RNA-Seq](#rna-seq)
- [Statistics](#statistics)
- [Tabular data](#tabular-data)
- [Utilities](#utilities)
- [Variant identification and analysis](#variant-identification-and-analysis)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Containers

* [project-name](http://example.com/) - A short description ends with a dot.

## Data visualization

* [arvestad/alv](https://github.com/arvestad/alv) - A console-based alignment viewer. ![GitHub Repo stars](https://img.shields.io/github/stars/arvestad/alv?style=social)
* [ChartsCSS/charts.css](https://github.com/ChartsCSS/charts.css) - Open source CSS framework for data visualization. ![GitHub Repo stars](https://img.shields.io/github/stars/ChartsCSS/charts.css?style=social)
* [cytoscape/cytoscape.js](https://github.com/cytoscape/cytoscape.js) - Graph theory (network) library for visualisation and analysis.
* [ryanlayer/samplot](https://github.com/ryanlayer/samplot) - Plot structural variant signals from many BAMs and CRAMs.

## Drawing

* [jgraph/drawio](https://github.com/jgraph/drawio) - A configurable diagramming application.

## Genome annotation

* [ncbi/pgap](https://github.com/ncbi/pgap) - NCBI Prokaryotic Genome Annotation Pipeline.

## Genome assembly

* [project-name](http://example.com/) - A short description ends with a dot.

## GWAS

* [MareesAT/GWA_tutorial](https://github.com/MareesAT/GWA_tutorial) - A comprehensive tutorial about GWAS and PRS.

## Images

* [faressoft/terminalizer](https://github.com/faressoft/terminalizer) - Record your terminal and generate animated gif images or share a web player.
* [flameshot-org/flameshot](https://github.com/flameshot-org/flameshot) - Powerful yet simple to use screenshot software.
* [nbedos/termtosvg](https://github.com/nbedos/termtosvg) - Record terminal sessions as SVG animations.
* [sindresorhus/pageres-cli](https://github.com/sindresorhus/pageres-cli) - Capture website screenshots.

## Machine learning

* [nidhaloff/igel](https://github.com/nidhaloff/igel) - A machine learning tool that allows you to train, test and use models without writing code.

## Metagenomics

* [project-name](http://example.com/) - A short description ends with a dot.

## Phylogenomics

* [project-name](http://example.com/) - A short description ends with a dot.

## Pipeline development

* [project-name](http://example.com/) - A short description ends with a dot.

## Programming

* [project-name](http://example.com/) - A short description ends with a dot.

## RNA-Seq

* [nf-core/nanoseq](https://github.com/nf-core/nanoseq) - Nanopore demultiplexing, QC and alignment pipeline.
* [nf-core/rnaseq](https://github.com/nf-core/rnaseq) - RNA sequencing analysis pipeline using STAR, RSEM, HISAT2 or Salmon with gene/isoform counts and extensive quality control.
* [STAR-Fusion/STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion) - Uses the STAR aligner to identify candidate fusion transcripts supported by Illumina reads.
* [suhrig/arriba](https://github.com/suhrig/arriba) - Fast and accurate gene fusion detection from RNA-Seq data.

## Sequence data

* [lh3/seqtk](https://github.com/lh3/seqtk) - Toolkit for processing sequences in FASTA/Q formats.

## Statistics

* [paulvanderlaken/ppsr](https://github.com/paulvanderlaken/ppsr) - R implementation of Predictive Power Score.
* [project-name](http://example.com/) - A short description ends with a dot.

## Structural variants

* [project-name](http://example.com/) - A short description ends with a dot.

## Tabular data

* [johnkerl/miller](https://github.com/johnkerl/miller) - Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON.
* [pstaender/csv2md](https://github.com/pstaender/csv2md) - Converts CSV data to Markdown tables.
* [wireservice/csvkit](https://github.com/wireservice/csvkit) - A suite of utilities for converting to and working with CSV, the king of tabular file formats.

## Utilities

* [adrianlopezroche/fdupes](https://github.com/adrianlopezroche/fdupes) - FDUPES is a program for identifying or deleting duplicate files residing within specified directories.
* [alacritty/alacritty](https://github.com/alacritty/alacritty) - A cross-platform, GPU-accelerated terminal emulator.
* [Clipy/Clipy](https://github.com/Clipy/Clipy) - Clipboard extension app for macOS.
* [joh/when-changed](https://github.com/joh/when-changed) - Execute a command when a file is changed.
* [jonschlinkert/markdown-toc](https://github.com/jonschlinkert/markdown-toc) - API and CLI for adding a table of contents to a Markdown file.
* [stevenvachon/broken-link-checker](https://github.com/stevenvachon/broken-link-checker) - Find broken links within HTML.
* [tcort/markdown-link-check](https://github.com/tcort/markdown-link-check) - Check hyperlinks in Markdown text.

## Variant identification and analysis

* [project-name](http://example.com/) - A short description ends with a dot.